demos/generomak/plasma/plot_core_profiles.py

```diff
@@ -0,0 +1,66 @@
+
+# Copyright 2016-2021 Euratom
+# Copyright 2016-2021 United Kingdom Atomic Energy Authority
+# Copyright 2016-2021 Centro de Investigaciones Energéticas, Medioambientales y Tecnológicas
+#
+# Licensed under the EUPL, Version 1.1 or – as soon they will be approved by the
+# European Commission - subsequent versions of the EUPL (the "Licence");
+# You may not use this work except in compliance with the Licence.
+# You may obtain a copy of the Licence at:
+#
+# https://joinup.ec.europa.eu/software/page/eupl5
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the Licence is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the Licence for the specific language governing permissions and limitations
+# under the Licence.
+
+import numpy as np
+
+import matplotlib.pyplot as plt
+
+from cherab.core.math.samplers import sample1d_points
+from cherab.generomak.plasma.plasma import get_core_profiles_description
+
+profiles = get_core_profiles_description()
+
+# setup temperature plot
+_, ax_t = plt.subplots()
+ax_t.set_title("Species Core Temperature Profiles")
+ax_t.set_xlabel("Psin")
+ax_t.set_ylabel("eV")
+
+# setup density plot
+_, ax_n = plt.subplots()
+ax_n.set_yscale("log")
+ax_n.tick_params(axis='both', which='both', labelsize=20)
```

Comment:
I don't understand why the font size for the tick labels should be increased here.

Reply:
This is an artefact of my pure eyesight and IDE setting. removed